build.sbt

```diff
@@ -1,11 +1,30 @@
+inThisBuild(List(
+  organization := "com.virtuslab",
+  homepage := Some(url("https://github.com/VirtusLab/pretty-stacktraces")),
+  licenses := List("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
+  developers := List(
+    Developer(
+      "BarkingBad",
+      "Andrzej Ratajczak",
+      "andrzej.ratajczak98@gmail.com",
+      url("https://twitter.com/aj_ratajczak")
+    ),
+    Developer(
+      "romanowski",
+      "Krzysztof Romanowski",
+      "kromanowski@virtuslab.com",
+      url("https://twitter.com/RomanowskiKr")
+    )
+  )
+))
+
 val scala3Version = "3.0.0-RC2"
 
 lazy val root = project
   .in(file("."))
   .settings(
     organization := "com.virtuslab",
     name := "pretty-stacktraces",
-    version := "0.1.0",
```

Comment:
You could also have `0.1.0-SNAPSHOT` or similar so it's easier to release locally

Reply:
Fixed